src/Schema/Directives/BaseDirective.php

```diff
@@ -125,6 +125,25 @@ protected function getModelClass(string $argumentName = 'model'): string
         return $this->namespaceModelClass($model);
     }
 
+    /**
+     * Find the model instance by the given key.
+     *
+     * @param string|int|null $id
+     * @return \Illuminate\Database\Eloquent\Model|null
+     * @throws \Nuwave\Lighthouse\Exceptions\DirectiveException
+     */
+    protected function findModelById($id = null): ?Model
```

Comment:
Why not just use `$modelClass::find()` here?

Reply:
You are right! I first passed in the whole array of arguments. Now I'm directly passing in the ID `find` is indeed better